src/Application/Regulation/View/Measure/MeasureView.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Regulation\View\Measure;
+
+use App\Application\Regulation\View\DailyRangeView;
+use App\Application\Regulation\View\PeriodView;
+use App\Application\Regulation\View\TimeSlotView;
+use App\Application\Regulation\View\VehicleSetView;
+use App\Domain\Regulation\Measure;
+
+class MeasureView
+{
+    public function __construct(
+        public readonly string $uuid,
+        public readonly string $type,
+        public readonly ?iterable $periods = null,
+        public readonly ?VehicleSetView $vehicleSet = null,
+        public readonly ?int $maxSpeed = null,
+        public array $locations = [],
```

Comment:
Il est pas en readonly ?

```suggestion
        public readonly array $locations = [],
```